src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.search-form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.bar {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding: 24px 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-size: 16px;
+  line-height: 21.86px;
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
```

Comment:
Set styles in html tag, other elements will inherit it

Reply:
It doesn't work from html tag. Probably because
- Remember that inputs and other interactive elements don’t inherit font styles by default.